### ISSUE TITLE
help for trial --coverage option contains epytext

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5784]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5784|
|Type|defect|
|Created|2012-07-15 15:18:03Z|

The `trial --help` output shows epytext for the `--coverage` option:

```
      --coverage           Generate coverage information in the I{coverage} file
                           in the directory specified by the I{trial-temp}
                           option.
```

Attachments:

* [coverage-5784.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/c1d/c1d3a9140488c85d94b8b773a33365a66b41c8ec/6e5e2eec23680d836a9dd7b440f6afe86f07e162.patch) (698 bytes) - added by thijs on 2012-07-15 15:23:05Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5784 5784
type__defect defect
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__ 
time__1342365483000000 1342365483000000
changetime__1342544123000000 1342544123000000
version__None None
owner__thijs thijs
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

It also talked about a `trial-temp` option, which doesn't exist and should be `temp-directory` instead.

**Comment by twisted-trac** [comment]
|[<img alt="ralphm's avatar" src="https://avatars.githubusercontent.com/u/116741?s=50" width="50" height="50">](https://github.com/ralphm)<a name="note_2"></a>|@ralphm set owner to @thijstriemstra|
|-|-|

This looks good to me. Please merge (with a news file).

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [34870]) Apply coverage-5784.patch: Remove epytext markup from trial's --coverage option and correct a reference to the temp-directory option.

Author: thijs
Reviewer: ralphm
Fixes: [#5784](https://github.com/twisted/twisted/issues/5784)
